k8s/logs-spack-io/certificates.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: tls-www
```

Comment:
need to add the namespace here

Reply:
Also, can we switch the name to something like `tls-logs`, to more easily distinguish it from the certs used for spack.io?

Reply:
yep!